plugins/ComposeAuth/README.md

```diff
@@ -0,0 +1,37 @@
+# Supabase-kt ComposeAuth
+
+Extends GoTrue-kt with composable
+
+# Installation
+
+
+Install plugin in main SupabaseClient. See the [documentation](https://supabase.com/docs/reference/kotlin/initializing) for more information
+```kotlin
+val client = createSupabaseClient(
+    supabaseUrl = "https://id.supabase.co",
+    supabaseKey = "apikey"
+) {
+    //...
+    install(GoTrue){
+        //your config
+    }
+    install(ComposeAuth) {
+        googleLoginConfig = googleNativeLogin(serverClientId = "google-client-id")
+        appleLoginConfig = appleNativeLogin()
+    }
+}
+```
+
+# Usage
+
+The composable can be accessed trough `composeAuth` property from `client`
+
+```kotlin
+val action = client.composeAuth.rememberLoginWithGoogle(onResult = {})
```

Comment:
somewhere here should maybe be explained how this works e.g. that Android uses Native login for Google, but on other platforms it defaults to `gotrue.loginWith(Google)` and explain that this is the fallback option. For `onResult` maybe there should be some indication that this is for error handling